tests/run-test-nbodykit.sh

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
+
+source testfunctions.sh
+
+FASTPM="`dirname $0`/../src/fastpm -T 1"
+mpirun -n 4 $FASTPM nbodykit.lua | tee log || fail
```

Comment:
I guess it's just a temporary file, but would it not be nicer to use a more descriptive filename, as opposed to just 'log'?

Reply:
I switch to creating a file with mktemp. Since we make assertions on the key features of the log file, there is probably no need to print it neither.